service/src/main/java/com/fecklessweasel/service/objectmodel/Department.java

```diff
@@ -1,30 +1,58 @@
 package com.fecklessweasel.service.objectmodel;
 
 import java.sql.Connection;
+import com.fecklessweasel.service.datatier.UniversityTable;
+import java.sql.Connection;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import com.fecklessweasel.service.UniversityUtil;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-
 import com.fecklessweasel.service.datatier.DepartmentTable;
 
 /**
  * Stores all information about a school's department.
  * @author Elliot Essman
- */
+<<<<<<< HEAD
+*/
 public final class Department {
-
-    /** ID in the database table. */
-    private int id;
-    /** The university this department is in. */
+    /**
+     * ID in the database table.
+     */
+    public static int id;
```

Comment:
Making this static breaks the object. If we have more than one Department loaded at one time, they will both point to the same object. (Static == all items have the same value).